_raml/apps/products/products.raml

```diff
@@ -316,34 +289,41 @@ post:
               application/json:
                 example: !include examples/put_product_productid_slideshow_sequence_response.json
                 schema: !include schema/put_product_productid_slideshow_sequence_schema.json
-  /custom-attributes:
+
+  /variations:
+    displayName: Product variations
     get:
       is: [ locale ]
-      description: Returns values of user-defined product attributes. *epages6 *epagesNow
+      queryParameters:
+        page:
+          description: Represents the current page.
+          type: integer
+          minimum: 1
+          default: 1
+        resultsPerPage:
+          description: Specifies the maximum number of results per page.
+          type: integer
+          default: 100
+          maximum: 1000
+      description: Returns a list of product variations.<br/>
+        The response contains links to the listed product variations and the properties `name` and `value` that were used are used for creating those variations.
```

Comment:
```suggestion
        The response contains links to the listed product variations and the properties `name` and `value` that were used for creating those variations.
```
Also not sure if "properties" is correct here, aren't they "attributes"...?

Reply:
@ywolpers You are completely right, since also "properties" is used for Beyond and we call them "attributes" in Now. And they are called "attributes" in a non-API context, so this fits for both scenarios :) Thanks for catching that :)